### ISSUE TITLE
`folded_description()` does not recognise the literal block style (`|`), so a literal-style description is measured as one character and the 1024-char gate passes without ever measuring anything

### ISSUE THREAD

**Issue by Pfeifdanix** [rebelytics/one-skill-to-rule-them-all#170]
## Environment

- `scripts/validate-skill-bundle.py`, `folded_description()` and the
  description check in `check_dir()`
- Verified at upstream `main` (`7518a85`); the function is identical in
  the local v3.1.0-era install it was found on
- macOS, `python3` 3.9 (system) without PyYAML, and a venv with
  PyYAML 6.0.3 — both measured, because the branch matters here

## What happened

A skill whose frontmatter opens `description: |` passed the pre-delivery
gate with `OK: all gate checks passed`. The description was in fact 444
folded characters, comfortably inside the cap — but the gate had not
measured it. It had measured the string `|`, one character long, which
passes the 1024 cap and the ~900 soft warning by construction. A skill
with a literal-style description of any length, including one well over
the cap, passes this gate unnoticed.

`folded_description()` matches only the folded block style:

```python
m = re.search(r"(?ms)^description:\s*>-?\s*\n(.*?)(?=^\S|\Z)", fm)
```

`>` and `>-` are covered; `|`, `|-`, `|+` and `>+` are not. When the
pattern misses, control falls through to the single-line branch, which
takes everything after `description:` on that one line — for a block
scalar, that is the indicator character itself.

## Reproduction

```bash
mkdir -p demo-skill
printf -- '---\nname: demo-skill\ndescription: |\n  Use when a staged skill needs a demonstration of a literal-block description\n  that the installer folds into a single value the gate is supposed to measure.\n---\n\n# Demo\n' > demo-skill/SKILL.md
python3 scripts/validate-skill-bundle.py demo-skill; echo "exit=$?"
```

| Instrument | Verdict |
|---|---|
| gate at `7518a85`, `python3` without PyYAML | `OK: all gate checks passed`, exit 0 |
| gate at `7518a85`, `python3` **with** PyYAML 6.0.3 | `OK: all gate checks passed`, exit 0 |
| what the gate measured | **1** char — `'|'` |
| `yaml.safe_load(fm)["description"]` | **154** chars |
| same gate, a `>-` folded description | `OK`, exit 0 — correctly measured |

The folded-style control passes in both versions, so the fixture is
testing the block style and nothing else. On a real skill
(`description: |`, 444 folded chars) the same run reports 1 against
PyYAML's 445 — the one-character difference is the trailing newline the
literal style preserves, which matters for nothing here but is stated
rather than smoothed over.

## How this differs from #158

Both touch `folded_description()`, and the titles look close enough that
this could be read as a restatement. It is not, and the difference is
measurable rather than editorial:

- **#158 is about which branch runs.** Without PyYAML the frontmatter is
  checked by pattern instead of by a parser, and an invalid single-line
  wrap slips through. Install PyYAML and that defect is caught.
- **This is about what the length check reads.** `desc =
  folded_description(fm)` sits outside the `try`/`except` and is called
  unconditionally, so the description length is measured by the regex in
  both modes. Installing PyYAML changes `data`, not `desc` — the row
  above shows the gate still reporting a pass with PyYAML present.
- **#158's fixes do not reach it.** Failing item (4) in the ImportError
  branch, adding a structural floor to the fallback, and planting a
  malformed input all address parse validity. The input here is
  *perfectly valid YAML*; the gate simply reads the wrong span of it.

They share #28's lesson — a gate that checks by text shape rather than by
the consumer's parser passes what the consumer would reject — from two
different directions.

## Suggested fix

1. **Recognise both block styles with any chomping indicator**, in
   `folded_description()`:

   ```python
   m = re.search(r"(?ms)^description:\s*[>|][-+]?\s*\n(.*?)(?=^\S|\Z)", fm)
   ```

   Newlines fold to spaces for both styles: the cap applies to the value
   the installer consumes, not to the source formatting.

2. **Print the measured value, not only the verdict.** The check
   currently prints nothing on success and a warning only above ~900, so
   a pass over a one-character read is indistinguishable from a real
   pass:

   ```python
   else:
       print(f"description: {len(desc)} chars (cap {MAX_DESCRIPTION_CHARS})")
   ```

   This is what surfaces the defect class rather than this one instance:
   a reader who sees `description: 1 chars` asks a question; a reader who
   sees `OK` does not.

3. **Add both block styles to the gate's own fixtures**, alongside the
   folded control, so a future refactor of the regex cannot silently drop
   one again.

The skill's own text already states the principle this violates — "an
unasserted metric does not merely miss defects, it manufactures
confidence that none exist" (`skill-authoring.md`, pre-delivery gate).
Here the metric is asserted; it is the measurement feeding it that is
empty.

## Principle

A check must report the value it measured, not only its verdict. A gate
that answers "passed" over a measurement that silently read one character
is indistinguishable from a gate that genuinely measured — and the
failure is asymmetric: the wrong instrument agreeing with you is never
investigated, so a right answer from a broken measurement teaches the
reader to trust it.

**Comment by rebelytics** [comment]
Fixed in v3.3.0. `folded_description()` handles the literal block style too, so a `|` description is no longer measured as one character. Thank you very much.
